tests/transport/test_requests.py

```diff
@@ -14,18 +14,27 @@
 
 import datetime
 import functools
+import os
 
 import freezegun
 import mock
+import OpenSSL
 import pytest
 import requests
 import requests.adapters
 from six.moves import http_client
 
 import google.auth.credentials
+import google.auth.transport._mtls_helper
 import google.auth.transport.requests
 from tests.transport import compliance
 
+DATA_DIR = os.path.join(os.path.dirname(__file__), "..", "data")
+with open(os.path.join(DATA_DIR, "privatekey.pem"), "rb") as fh:
+    PRIVATE_KEY_BYTES = fh.read()
+with open(os.path.join(DATA_DIR, "public_cert.pem"), "rb") as fh:
+    PUBLIC_CERT_BYTES = fh.read()
+
```

Comment:
Since the key and cert are being used in multiple fields it may be worth making a `conftest.py` to avoid duplicating code.

Reply:
moved to conftest.py